tools/cask/main.go

```diff
@@ -88,10 +88,10 @@ func Update(image string, dockerClient *client.Client, ctx context.Context) {
 	out, err := dockerClient.ImagePull(ctx, image, types.ImagePullOptions{ })
 	if err != nil {
 		log.Printf( "Image Pull failed, will use cached image if available. %v", err)
```

Comment:
should we log fatal and be done here? (ie exit).

Reply:
It is OK if it fails.  You just won't have the latest but at least you can keep working if the registry is down (assuming you previously successfully pulled).   During dev I put a local image ID into the `CUSTODIAN_IMAGE` environment variable which it fails to pull, but it still works fine then.

Reply:
sounds good